Setup/Uninstall.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
```

Comment:
1.3.1. All new PHP files MUST have strict type mode enabled by starting with declare(strict_types=1);. All updated PHP files SHOULD have strict type mode enabled. PHP interfaces MAY have this declaration.

https://developer.adobe.com/commerce/php/coding-standards/technical-guidelines/#1-basic-programming-principles